backend/src/routes/auth.py

```
@@ -0,0 +1,59 @@
+from flask import Blueprint, request, jsonify
+from flask_jwt_extended import create_access_token
+from src.models.database import db, User, Tenant
+from werkzeug.security import generate_password_hash, check_password_hash
```

Ruff (0.11.9)

`werkzeug.security.generate_password_hash` imported but unused

Remove unused import

(F401)